src/graphql/schema.js

```diff
@@ -47,92 +47,89 @@ enum _TransactionStatus {
 
 type Topic {
   txid: String!
-  version: Int!
   blockNum: Int
-  status: _OracleStatusType!
   address: String
-  escrowAmount: String
+  creatorAddress: String!
+  hashId: String
+  status: _OracleStatusType!
   name: String!
   options: [String!]!
-  resultIdx: Int
   qtumAmount: [String!]!
   botAmount: [String!]!
+  resultIdx: Int
+  escrowAmount: String
   oracles: [Oracle]
   transactions: [Transaction]
-  creatorAddress: String!
-  hashId: String
+  version: Int!
 }
 
 type Oracle {
   txid: String!
-  version: Int!
   blockNum: Int
-  status: _OracleStatusType!
   address: String
   topicAddress: String
-  resultSetterAddress: String
```

Comment:
what's the difference between `resultSetterAddress` and `resultSetterQAddress`, and what's the new `resultSetterAddress`?

Reply:
I removed all `...QAddress` fields since we are storing both as the Qtum address format. eg. `resultSetterAddress` and `resultSetterQAddress` was the same Qtum address.